packages/workflow/docs/Workflow.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { Button } from "@kaizen/button"
+import { ComponentDocsTemplate } from "../../../storybook/components/DocsContainer"
+import { Workflow, WorkflowExit, WorkflowProps } from "../"
+
+const defaultArgs = {
+  workflowName: "Create a self-reflection cycle",
+  stepName: "Settings",
+  steps: ["Settings", "Questions", "Preview", "Employees", "Schedule"],
+  isComplete: false,
+  status: {
+    content: "Draft",
+    vairant: "statusDraft",
+  },
```

Comment:
Something good to call out in a story would be that this extends the `Tag` component's variants, so you don't necessarily have to showcase every variant on this page, and they can go check relevant docs.